src/05-derivatives/05-diffusion-derivatives.md

```diff
@@ -161,307 +388,292 @@ Parameters that may be stored within the JSON sidecar "Parameters" field depend
+| `mode`              | Mode of the tensor                                                     | [Scalar](#scalar)       | { `dki`, `dti`, `fwdti`, `wmti` }               |                                                                |
+| `msd`               | Mean-Squared Displacement                                              | [Scalar](#scalar)       | { `mapmri`, `shore` }                           |                                                                |
+| `pdf`               | Diffusion propagator                                                   | [3-vectors](#3-vectors) |                                                 |                                                                |
+| `peak`              | Direction(s) and amplitude(s) of ODF maximum (maxima)                  | [3-vectors](#3-vectors) | { `csa`, `csd`, `forecast`, `shore` }           | Same units as ODF                                              |
+| `rd`                | Radial Diffusivity (also called perpendicular diffusivity)             | [Scalar](#scalar)       | { `dki`, `dti`, `forecast`, `fwdti`, `wmti` }   | \mu m<sup>2</sup>.ms<sup>-1</sup> <sup>[1](#diffusivity)</sup> |
+| `rk`                | Radial kurtosis                                                        | [Scalar](#scalar)       | { `dki`, `wmti` }                               | Unitless                                                       |
+| `rtap`              | Return To Axis Probability                                             | [Scalar](#scalar)       | { `mapmri` }                                    | Probability \[0.0-1.0\]                                        |
+| `rtop`              | Return To Origin Probability                                           | [Scalar](#scalar)       | { `shore` }                                     | Probability \[0.0-1.0\]                                        |
+| `rtpp`              | Return To Plane Probability                                            | [Scalar](#scalar)       | { `mapmri` }                                    | Probability \[0.0-1.0\]                                        |
+| `tort`              | Tortuosity of extra-cellular space                                     | [Scalar](#scalar)       | { `dki` }                                       |                                                                |
+
+While not explicitly included in the table above, *any* [Scalar](#scalar)
+extrinsic parameter can theoretically be combined with a separate source
+of orientation information from the diffusion model in order to produce a
+[directionally-encoded colours (DEC)](#directionally-encoded-colours),
+[spherical coordinates](#spherical-coordinates) or
+[3-vectors](#3-vectors) image.
+
+## Demonstrative examples
+
+-   A basic Diffusion Tensor fit:

```

Comment:
This is great, but I think we should put it above

Reply:
Similar to some previous comments:

I've tried to make this more of a "specification" document and less of a "tutorial" document. Since many of the examples draw from content throughout the entirety of the rest of the document, it would be incomplete to present such prior to any of that content. If the slightly technical / general descriptions are confusing to a reader, they can simply "skip" to the end of the document to see the examples; I don't think it's our role here to hold the reader's hand closely as they try to familiarise themselves by reading through the document in a linear fashion. If there's a specific point that feels like it currently requires a demonstrative example to convey, we should probably look at it more closely; either improving the explanation, or with a minimal example that demonstrates just that one specific idea.